### PR TITLE
Add activity for proposal unscheduled transition.

### PR DESCRIPTION
**Pull request by njohner**
There was no ativity defined for the transition 'scheduled-submitted'. Instead, the notification sent to the issuer when removing a proposal from a meeting schedule was the same as when the proposal got scheduled.

**personal-preferences view with the added setting**
screen shot 2018-08-29 at 16 00 40

**correct message is now displayed in the notifications**
screen shot 2018-08-29 at 16 01 54

### CODE REVIEW

File: opengever/activity/__init__.py [hunk 1]
```diff
@@ -151,6 +153,7 @@ def send_digest_zopectl_handler(app, args):
 ACTIVITIES_ICONS = {
     'proposal-transition-reject': 'rejected',
     'proposal-transition-schedule': 'scheduled',
+    'proposal-transition-unschedule': 'unscheduled',
```
Comment (deiferni) [inline opengever/activity/__init__.py]: this causes a naming conflict with proposals that are not yet scheduled, we call them unscheduled, too. i would not use unscheduled in another context for proposals. what do you think about deschedule 🤔?

Comment (Rotonen) [reply opengever/activity/__init__.py]: How about 'pulled'?

Comment (deiferni) [reply opengever/activity/__init__.py]: as just discussed `pulled` seems to be used in meetings and is unique enough. so we'll go for it.

**Comment by njohner** [comment]
Ok this should be ready to ship. I tested the upgrade again after the change in naming and everything seems to still work.

**Pull request merged**